src/main/java/org/wso2/config/mapper/ConfigConstants.java

```diff
@@ -37,4 +37,6 @@ private ConfigConstants() {
     static final String SYSTEM_PROPERTY_PREFIX = "sys:";
     static final String ENVIRONMENT_VARIABLE_PREFIX = "env:";
 
+    static final String AVOID_RESOLVING_ENV_AND_SYS_VARIABLE = "avoidResolvingEnvAndSysVariable";
```

Comment:
I feel like the plural form avoidResolvingEnvAndSysVariable**s** sounds better. WDYT?

Reply:
Done in 4164eb605ede061f27b161aaca0afb30d0864262